h2o-py/tests/testdir_algos/xgboost/pyunit_xgboost_pojo.py

```diff
@@ -0,0 +1,107 @@
+import sys
+import os
+sys.path.insert(1, "../../../")
+from h2o.estimators.xgboost import *
+from tests import pyunit_utils
+
+
+def compare_preds(train, test, x, y, booster, ntrees, max_depth, max_error):
```

Comment:
this is good, are there enough `testJavaScoring` tests in XGBoostTest to cover comparison of mojo / pojo / h2o?

Reply:
it actually discovered some bugs in the pojo, so yes, there is good coverage there as well (especially for gbtree)